Sources/PersistentCollections/PersistentDictionary+Codable.swift

```diff
@@ -0,0 +1,173 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+// Code in this file is a slightly adapted variant of `Dictionary`'s `Codable`
+// implementation in the Standard Library as of Swift 5.7.
+// `PersistentDictionary` therefore encodes/decodes itself exactly the same as
+// `Dictionary`, and the two types can each decode data encoded by the other.
```

Comment:
The according tests in https://github.com/lorentey/swift-collections/blob/PersistentCollections-updates/Tests/PersistentCollectionsTests/PersistentCollections%20Tests.swift#L970-L1033 should preferably be activated upon the time the feature is added.